synapseAnnotations/data/compoundScreen.json

```diff
@@ -0,0 +1,72 @@
+[
+  {
+    "name": "compoundName",
```

Comment:
@allaway Can you please add the drug names/sources you provided for me in json format?

Reply:
@sgosline yes - how do you want me to resolve the cases where the source compound name is different from the name used in the data?